src/qobj/quantum_object.jl

```diff
@@ -369,6 +380,8 @@ SparseArrays.SparseMatrixCSC(A::QuantumObject{<:AbstractMatrix}) =
 SparseArrays.SparseMatrixCSC{T}(A::QuantumObject{<:SparseMatrixCSC}) where {T<:Number} =
     QuantumObject(SparseMatrixCSC{T}(A.data), A.type, A.dims)
 
+get_typename_wrapper(A::AbstractQuantumObject) = Base.typename(typeof(A)).wrapper
```

Comment:
how about moving this function into `utilities.jl`, since it would be used by both `quantum_object.jl` and `quantum_object_evo.jl`